README.md

```diff
@@ -41,10 +41,18 @@ The priority is applied as follows:
 
 The order of objects with equal (implicit) priority is undefined.
 
+## GlobalTracer
+
+If the [opentracing-util] library is detected and a [`GlobalTracer`][globaltracer] 
```

Comment:
@bhs Thanks for the review (esp. with the little one taking your time). Glad you agree with the idea.